audioipc/src/rpc/server.rs

```diff
@@ -61,7 +61,7 @@ where
     };
 
     // Spawn the RPC driver into task
-    handle.spawn(Box::new(fut.map_err(|_| ())))
+    current_thread::spawn(fut.map_err(|_| ()))
```

Comment:
I notice that several `handle`s (and `handle` parameters in the function) are replaced by `current_thread`s. Will the threading model works in the same way as usual by these changes?

Reply:
The threading model remains the same for this code as it was with tokio-core.